### ISSUE TITLE
the argument "labels" is no longer used in tf.nn.sigmoid_cross_entropy_with_logits in tensorflow v0.12.1

### ISSUE THREAD

**Issue by umbriel47** [carpedm20/DCGAN-tensorflow#87]
"labels" should be changed to "targets" is this is used with tf v0.12.1

**Comment by ppwwyyxx** [comment]
Actually "targets" is no longer used after 12.1.
See https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/api_docs/python/functions_and_classes/shard5/tf.nn.sigmoid_cross_entropy_with_logits.md .

**Comment by carpedm20** [comment]
That's right. I'll close this issue then.
